packages/app-page-builder/src/editor/components/MediumEditor/index.ts

```diff
@@ -17,6 +17,7 @@ interface ReactMediumEditorProps {
     onSelect: (value: string) => void;
     tag: string | [string, Record<string, any>];
     options?: CoreOptions;
+    forceFocus?: boolean;
```

Comment:
Please rename this to `autoFocus`, as we already use that prop name in other components. Just to be consistent.